velox/connectors/hive/HiveConnectorUtil.h

```diff
@@ -108,4 +108,10 @@ core::TypedExprPtr extractFiltersFromRemainingFilter(
     SubfieldFilters& filters,
     double& sampleRate);
 
+void fromHiveConfigToWriterOptions(
```

Comment:
```
s/fromHiveConfigToWriterOptions/updateWriterOptionsFromHiveConfig/
```

Can you briefly comment this utility and add unit test to cover all the existing conversions? Thanks!